sentinel-core/src/main/resources/META-INF/services/com.alibaba.csp.sentinel.event.subscriber.EventSubscriber

```diff
@@ -0,0 +1,2 @@
+# Default event subscriber
+com.alibaba.csp.sentinel.event.subscriber.DefaultEventSubscriber
```

Comment:
Two same file in `META-INF/services`, and the package of test case `DefaultEventSubscriberTest` is `com.alibaba.csp.sentinel.event`.